docs/content/howto/callbacks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: React to events in the Viewer
+order: 50
+---
+
+We support registering callbacks to Viewer events in these environments:
+
+- Web browsers, through our [JS package](./integrations/embed-web.md#callbacks)
+- Jupyter notebooks, through our [Notebook API](./integrations/embed-notebooks.md#reacting-to-events-in-the-viewer)
+
```

Comment:
this page feels a bit naked, but the referenced ones are quite good imho. Wondering if we can write some more generalities, future plans and how that's useful 🤔 

[not super actionable right now, we can always add more stuff later]